src/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -872,6 +872,61 @@ internal string GetPropertyRawText()
             return _parent.GetPropertyRawValueAsString(_idx);
         }
 
+        /// <summary>
+        ///   Write the element into the provider writer as a named object property.
```

Comment:
nit: `... provided writer`

Reply:
I'm not sure what your suggestion here is.  I'm pretty sure that the "the" in "the provided writer" is correct.

Reply:
>  Write the element into the provider writer as a named object property.

`provider` -> `provided`